pkg/Build/main.js

```diff
@@ -35,6 +37,11 @@ export const main = async (xenon, App, Composer) => {
   Design.configureDesignApp({xenon, Composer, customLibraries});
   const {nodeTypes, services} = await loadLibraries(customLibraries);
   persistations.$NodeTypeList$typeList$nodeTypes = nodeTypes;
+  persistations.$GraphList$graphAgent$publishedGraphsUrl = `${globalThis.config.firebaseConfig.databaseURL}/${globalThis.config.publicGraphsPath}`;
+  const buildGraph = await loadGraph(Params.getParam('graph'));
```

Comment:
i'm not so sure when i should clear this param, if at all:
- atm the param remains, even when a different graph is selected from the list
- when i select another graph, the local storage `selectedMeta` is updated (but not when loading first time)...

not so sure what the correct behaviour should be overall, but i think this is still a step in the right direction (and i can add a link from site to load particular graphs in Build)

Reply:
I'm not sure either, let's try it on and see.